content/en/docs/concepts/configuration/configmap.md

```diff
@@ -88,7 +88,7 @@ data:
 There are four different ways that you can use a ConfigMap to configure
 a container inside a Pod:
 
-1. Command line arguments to the entrypoint of a container
+1. Inside a Pod command
```

Comment:
According to the resource definition, we don't have a `cmd` property for a Pod, right?
I guess you were trying to say "Inside a container command"?

Reply:
Hi @tengqm, thanks for the review. 
That's correct.
I have updated.

Initially I wanted to be consistent with:  https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#use-configmap-defined-environment-variables-in-pod-commands

> You can use ConfigMap-defined environment variables in the command section of the Pod specification using the $(VAR_NAME) Kubernetes substitution syntax.

Therefore I also updated that part in https://github.com/kubernetes/website/pull/25089/commits/